src/SFA.DAS.Roatp.CourseManagement.Domain/ApiModels/ProviderLocation.cs

```diff
@@ -7,8 +7,13 @@ public class ProviderLocation
         public Guid NavigationId { get; set; }
         public int? RegionId { get; set; }
         public string LocationName { get; set; }
+        public string AddressLine1 { get; set; }
```

Comment:
When we get list of locations, does it include all these fields? if not, then we should have separate models.